assets/src/pages/room/components/StreamPlayer/ScreenSharingPlayers.tsx

```diff
@@ -17,9 +17,9 @@ type Props = {
 const ScreenSharingPlayers: FC<Props> = ({ streams }: Props) => {
   return (
     <div className="h-full mb-3 md:mr-3 md:mb-none active-screensharing-grid grid-cols-1 md:grid-cols-1">
-      {streams.map((config, idx) => (
+      {streams.map((config) => (
         <MediaPlayerTile
-          key={`${idx}${config.video.stream?.id ?? ""}`}
+          key={config.video.stream?.id}
```

Comment:
`ID` in `TrackWithId` could be optional because it represents the track id from the server. It is possible to start screen sharing without invoking`addTrack` function. In this implementation (`key={config.video.stream?.id}`) I assumed that if we stop sharing the screen we'll set stream to null and therefore we'll remove this component from DOM and id would never be undefined. 

But I found a simpler approach. I've added `mediaPlayerId` field to the `config` object. For the local screen-sharing stream I set that to LOCAL_SCREEN_SHARING_ID constant and for every remote track that ID is set to remote track ID.